core/templates/app.constants.ajs.ts

```diff
@@ -108,28 +108,18 @@ angular.module('oppia').constant(
 angular.module('oppia').constant(
   'COMPONENT_NAME_CONTENT', AppConstants.COMPONENT_NAME_CONTENT);
 angular.module('oppia').constant(
-  'COMPONENT_NAME_HINT', AppConstants.COMPONENT_NAME_HINT);
-angular.module('oppia').constant(
-  'COMPONENT_NAME_SOLUTION', AppConstants.COMPONENT_NAME_SOLUTION);
+  'COMPONENT_NAME_EXPLANATION', AppConstants.COMPONENT_NAME_EXPLANATION);
```

Comment:
I'm not sure what this refers to. If it's the subpart of solution, should it be SOLUTION_EXPLANATION rather than EXPLANATION?